src/test.rs

```diff
@@ -31,8 +31,36 @@ fn binary_from_keypair(e: &Env, kp: &Keypair) -> U256 {
     bin.try_into().unwrap()
 }
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+struct ExternalBigInt(num_bigint::BigInt);
```

Comment:
Why is `num_bigint` required? Why does it need to be wrapped in `ExternalBigInt`?

Reply:
Presumedly any contract is likely to need to do this, got some thoughts on how to codify this pattern into the SDK?

Reply:
I don't yet--I consider what I did here a kludge. But this is like making an "external" function call, so when I work on testing multiple contracts simultaneously I hope to discover a general solution.